### PR TITLE
chore: add atomic cli unit & e2e tests, add atomic documentation and prepare for release

### PR DESCRIPTION
**Pull request by ThibodeauJF**
## Testing

- [x] Unit Tests:

- [x] Functionnal Tests:

- [x] Manual Tests:

https://coveord.atlassian.net/browse/CDX-669
https://coveord.atlassian.net/browse/CDX-698
https://coveord.atlassian.net/browse/CDX-699

### CODE REVIEW

File: packages/create-atomic/templates/README.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+This project was generated with [@coveo/create-atomic](https://npmjs.com/package/@coveo/create-atomic).
+
+## Setup environment
+
+The root folder should contain a `.env` file. Replace all placeholder variables (`<...>`) by the proper information for your organization. Consult the example configuration file named `.env.example` as needed. For more involved configurations, you can modify the request parameters used in the `lambda/functions/token/token.ts` file.
+
+### CDN
+
+By default, the project installs the latest major Atomic version, v1, to allow types and more advanced customizations. [Coveo Headless](https://www.npmjs.com/package/@coveo/headless) is also bundled with Atomci and accessible at `@coveo/atomic/headless`
```
Comment (louis-bompart) [inline packages/create-atomic/templates/README.md:11]:
```suggestion
By default, the project installs the latest major Atomic version, v1, to allow types and more advanced customizations. [Coveo Headless](https://www.npmjs.com/package/@coveo/headless) is also bundled with Atomic and accessible at `@coveo/atomic/headless`
```

**Review by louis-bompart — APPROVED** [review]
LGTM, I would check with @jpmarceau because lot of text

**Review by olamothe — APPROVED** [review]
👍 Really nice !

File: packages/create-atomic/README.md [hunk 2]
```diff
@@ -24,3 +24,5 @@ or
 npm install -g @coveo/create-atomic
 create-atomic myapp
 ```
+
+Running the project directly, you will be prompted in order to fill out your organization's authentication information.
```
Comment (jpmarceau) [inline packages/create-atomic/README.md:28]:
```suggestion
If you run the project directly, you will be prompted to fill out your organization's authentication information.
```

File: packages/create-atomic/templates/README.md [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+This project was generated with [@coveo/create-atomic](https://npmjs.com/package/@coveo/create-atomic).
+
+## Setup environment
+
+The root folder should contain a `.env` file. Replace all placeholder variables (`<...>`) by the proper information for your organization. Consult the example configuration file named `.env.example` as needed. For more involved configurations, you can modify the request parameters used in the `lambda/functions/token/token.ts` file.
+
+### CDN
+
+By default, the project installs the latest major Atomic version, v1, to allow types and more advanced customizations. [Coveo Headless](https://www.npmjs.com/package/@coveo/headless) is also bundled with Atomic and accessible at `@coveo/atomic/headless`
```
Comment (jpmarceau) [inline packages/create-atomic/templates/README.md:11]:
```suggestion
By default, the project installs the latest major Atomic version, v1, to allow types and more advanced customizations. [Coveo Headless](https://www.npmjs.com/package/@coveo/headless) is also bundled with Atomic and accessible at `@coveo/atomic/headless`.
```

**Review by jpmarceau — APPROVED** [review]
👌

**Review by louis-bompart — APPROVED** [review]
image

File: packages/cli/src/commands/ui/create/atomic.ts [hunk 4]
```diff
@@ -38,11 +39,9 @@ export default class Atomic extends Command {
...
       char: 'v',
       description: `The version of ${Atomic.cliPackage} to use.`,
-      // default: getPackageVersion(Atomic.cliPackage), TODO: uncomment when @coveo/create-atomic added to package.json
-      default: 'latest',
+      default: getPackageVersion(Atomic.cliPackage),
     }),
   };
-  public static hidden = true;
 
   @Trackable()
```
Comment (y-lakhdar) [inline packages/cli/src/commands/ui/create/atomic.ts:46]:
```suggestion
  @Trackable({
    eventName: 'ui create',
    overrideEventProperties: {framework: 'atomic'},
  })
```

This is a small change to ensure analytic data is consistent in Amplitude. For more info consult [CLI Taxonomy](https://docs.google.com/spreadsheets/d/1lexw5Nmg0qmtwaHHtBydDlJxllS_K5xSeigE4z3pGDk/edit#gid=771066228).

**Review by y-lakhdar — APPROVED** [review]
Well done

**Pull request merged**